test/distributed/org/apache/cassandra/distributed/shared/ClusterUtils.java

```diff
@@ -187,6 +187,24 @@ public static <I extends IInstance> void stopAll(ICluster<I> cluster)
         cluster.stream().forEach(ClusterUtils::stopUnchecked);
     }
 
+    /**
+     * Restart an instance in a blocking manner.
+     * <p>
+     * This method stops the instance using {@link #stopUnchecked(IInstance)} and then
```

Comment:
I feel like this comment takes more time to read than the code. I do not mind the short "Restart an instance in a blocking manner" part though.